examples/psort.cpp

```diff
@@ -8,28 +8,50 @@
 #include <parallel/algorithm>
 #include <fcntl.h>
 #include <omp.h>
+#include <cstdio>
+#include <cstring>
 #include "umap/umap.h"
 
 using namespace std;
 
-int main(int argc, char **argv)
+void initialize_sortfile( const char* fname, uint64_t totalbytes )
```

Comment:
Should this be called initialize_and_sort_file since it does both

Reply:
Agreed.  I changed it.